### PR TITLE
fix: Stop configuring logging

### PR DESCRIPTION
**Pull request by LordOfPolls**
## About

Stop configuring logging. 

## Checklist

- [x] I've ran `pre-commit` to format and lint the change(s) made.
- [x] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues).
  - (If existent):
- [x] I've made this pull request for/as: (check all that apply)
  - [ ] Documentation
  - [x] Breaking change
  - [ ] New feature/enhancement
  - [ ] Bugfix

### CODE REVIEW

**Comment by V3ntus** [comment]
Handler is still being configured?
image

**Comment by LordOfPolls** [comment]
It shouldnt be, i cant see any additional references to the stream, uninstall discord-interactions and reinstall my pr. My bet is pip resolution is being fucky again

File: interactions/api/dispatch.py [hunk 1]
```diff
@@ -1,15 +1,11 @@
 from asyncio import get_event_loop
-from logging import Logger, StreamHandler, basicConfig, getLogger
+from logging import Logger, basicConfig, getLogger
 from typing import Coroutine, Optional
 
-from ..base import CustomFormatter, Data
+from ..base import Data
 
 basicConfig(level=Data.LOGGER)
```
Comment (benwoo1110) [inline interactions/api/dispatch.py]: basicConfig should also be removed? (i think)

Comment (LordOfPolls) [reply interactions/api/dispatch.py]: yep, im a moron and forgot that would also enable a stream

**Comment by LordOfPolls** [comment]
That should be all of them, ive run it, logging definitely isnt outputting anything

**Comment by V3ntus** [comment]
Hm. I'm on be2fdcb, and I'm sure I'm importing this repo/PR as a module as I've got no other instances of the package elsewhere outside of my python env. Still getting log output, oddly.

image

But if you confirm no output, then I trust you more, since I'm prone to missing things here because of my lack of skill.

**Comment by i0bs** [comment]
too lazy to actually merge this **cough** @DeltaXWizard

**Review by FayeDel — APPROVED** [review]
This looks good to me polls :)
(Though, I wonder why this is done in stable and not unstable)
If you want to change this you can if you want

(ly fl0w)

**Pull request merged**